lambdas/tests/unit/common/utils/request-utils.test.ts

```diff
@@ -0,0 +1,45 @@
+import { getClientIpAddress, getSessionId } from "../../../../src/common/utils/request-utils";
+
+describe("getClientIpAddress", () => {
+    test("returns the value of x-forwarded-for header", () => {
+        const result = getClientIpAddress({
+            headers: {
+                "x-forwarded-for": "192.168.1.1"
```

Comment:
Could we get a test for different casing of the header? e.g. `X-FORWARDED-FOR` - the spec says they should be case insensitive, and we're using a `lowercase` in the code, so it'd be good to test that option too.